src/inspectors/computeLocalHash.js

```diff
@@ -24,6 +27,11 @@ CONTEXTS['https://w3id.org/blockcerts/v2'] = BLOCKCERTSV2_CONTEXT;
 CONTEXTS['https://www.w3id.org/blockcerts/schema/2.0/context.json'] = BLOCKCERTSV2_CONTEXT;
 CONTEXTS['https://w3id.org/blockcerts/v1'] = BLOCKCERTSV1_2_CONTEXT;
 
+// V3
+CONTEXTS['https://www.blockcerts.org/schema/3.0-alpha/context.json'] = BLOCKCERTSV3_CONTEXT;
+CONTEXTS['https://www.w3.org/2018/credentials/v1'] = VERIFIABLE_CRED_CTX;
+CONTEXTS['https://www.w3.org/2018/credentials/examples/v1'] = VERIFIABLE_CRED_EXAMPLE;
```

Comment:
We may want to add `https://w3id.org/blockcerts/schema/3.0-alpha/merkleProof2019Context.json `/`https://www.blockcerts.org/schema/3.0-alpha/merkleProof2019Context.json`.

Also, adding the redirected context might be useful as well. Since `https://w3id.org/blockcerts/schema/3.0-alpha/context.json` redirects to `https://www.blockcerts.org/schema/3.0-alpha/context.json`.